src/core/Tooltip/index.tsx

```diff
@@ -1,39 +1,16 @@
-import {
-  makeStyles,
-  Tooltip as RawTooltip,
-  TooltipProps,
-} from "@material-ui/core";
+import { useTheme } from "@emotion/react";
+import { Tooltip as RawTooltip, TooltipProps } from "@material-ui/core";
 import React from "react";
-import { AppThemeOptions } from "../styles";
+import { arrowCss, tooltipCss } from "./style";
 
 export { TooltipProps };
 
-const useStyles = makeStyles((theme: AppThemeOptions) => {
```

Comment:
`makeStyles()` generates a css class name for each key (e.g., `arrow` and `tooltip`), so we can replace it with [`css()`](https://emotion.sh/docs/@emotion/css) from `import { css } from '@emotion/css`, which also generates a css class name as an output!

Note that there's ANOTHER `css()`, which is from `import { css } from '@emotion/react'`, which the output is serialized style instead of css class name. So we just need to be aware of when to use which, this confused me so much at the beginning 😂 

`themeToCss()` in `src/core/styles/common/mixins/fonts.ts` is an example of using `css()` from `@emotion/react`!